test/test-spawn.c

```diff
@@ -1394,6 +1394,64 @@ TEST_IMPL(spawn_no_path) {
   MAKE_VALGRIND_HAPPY(uv_default_loop());
   return 0;
 }
+
+
+TEST_IMPL(spawn_no_ext) {
+  char new_exepath[1024];
+
+  init_process_options("spawn_helper1", exit_cb);
+  memcpy(new_exepath, exepath, exepath_size - (sizeof(".exe") - sizeof(char)));
+  strcpy(new_exepath + exepath_size - (sizeof(".exe") / sizeof(char) - 1), "_no_ext");
```

Comment:
sizeof(char) is always 1 and use snprintf() instead strcpy():
```suggestion
  snprintf(new_exepath, sizeof(new_exepath), "%.*s_no_ext",
           (int) (exepath_size - sizeof(".exe") + 1),
           exepath);
```
Also in the next test.